web/src/components/Plugin/data.tsx

```diff
@@ -74,6 +74,15 @@ export enum PluginType {
   other = 'other',
 }
 
+export enum PluginState {
+  disable = 10,
+  enable = 20,
```

Comment:
Hi, can you explain why it is written like this?

Reply:
it was written in order to better extend the state, to display different logic and to facilitate sorting, for example 

- not added --> 0  
- added --> 1 
- disable --> 10 
- enable --> 20 

Sorting by state: enable > disable > added  > not added